### PR TITLE
add missing files

### PR DESCRIPTION
**Pull request by vonbecmann**
some files were left out of the commit.
for example:
.properties was not added
.filetree at the root of the source dir was not added.
there is something wrong with way the relative path is added to the index of the repository.
i tested once and it works in pharo 6.1. 
is it ok?

### CODE REVIEW

**Comment by estebanlm** [comment]
.filetree files need to be removed.
the absence of .properties indeed, should be there.

**Comment by vonbecmann** [comment]
yes, you are right. let me try again.

the removal of the .filetree at the root was not added to the index. (<- fail )
all other .filetree files were removed and the removal was added to the index. (<- ok)

there's something wrong between the relative paths and 
the methods addFilesToIndex:, addProperties:

thanks.
